### PR TITLE
Layout and javascript fixes

### PR DESCRIPTION
**Pull request by yalamber**
* UI updates 
* Added version and initial status informations

### CODE REVIEW

File: hornetctl/hornetctl.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            var proc = cockpit.spawn(['hornet_clean_db', '-m', '-c'], {superuser:"require"});
+            proc.done(function () {
+                window.alert('Database clean successfull')
+            })
+            proc.fail(hornetCtl.failHandler) 
+        }
+
+    },
+    snapshot: function () {
+        var check = confirm('Snapshot Database?');
```
Comment (bernardoaraujor) [inline hornetctl/hornetctl.js]: `Snapshot Database` is not an accurate description..
the purpose of this button is to `Clear Snapshot`, so I think that would be a more appropriate warning..

File: hornetctl/hornetctl.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        if(check) {
+            var proc = cockpit.spawn('hornet_clean_export', {superuser:"require"});
+            proc.done(function () {
+                window.alert('Snapshot successfull')
```
Comment (bernardoaraujor) [inline hornetctl/hornetctl.js]: and here: `Successfully cleaned Snapshot`

**Review by bernardoaraujor — CHANGES_REQUESTED** [review]
It's looking great! Thank you so much.
`Clean Snapshot` button warning needs to be fixed.

Do you know if it's possible to add a `Service Logs` box like the one you can see in `http://localhost:9090/system/services#/hornet.service`?

File: hornetctl/hornetctl.js (hunk 2 above)
Comment (yalamber) [reply hornetctl/hornetctl.js]: pushed fix for this

**Comment by yalamber** [comment]
Please review logs panel. I used system logs component https://cockpit-project.org/guide/latest/api-logs-html.html

**Comment by bernardoaraujor** [comment]
@yalamber logs panel looks beautiful 🤩
do you know why it asks to log in again?

Two things that would be great to add:
- `Restart` button next to `Start` and `Stop`
- `Launch` button for Dashboard section that opens `http://board.local.ip.addr:8081` on a new browser tab.

Here's the function definition for `Restart`:
```
var proc = cockpit.spawn(['systemctl', 'restart', 'hornet'], {superuser:"require"});
```

When it comes to `Launch` Dashboard button, I'm not sure how to actually implement it.
It's important to remember that in actual use cases, hostname is not `localhost` but the IP address of the board where honeycombOS is running. So the button should inherit the same hostname under which Cockpit is currently loaded.
Also, the button needs to show some kind of warning in case the Dashboard is disabled.

**Pull request merged**